pages/common/task.md

```diff
@@ -1,24 +1,36 @@
 # task
 
-> TODO list manager.
-> More information: <https://manned.org/task>.
+> Manage your TODO list from the command line.
```

Comment:
I prefer the old description.

Reply:
I somehow agree. "Manage **your** TODO list" sounds a little like it is a client for some service like `todoist`. We could change it to this tough:
```suggestion
> Command-line TODO list manager.
```

Reply:
Do we need to have TODO in caps?
Can't it be 
```suggestion
> Command-line to-do list manager.
```

Reply:
Sure, `to-do` works too. We just have to be consistent.